### DISCUSSION TITLE
Just a funny question

### DISCUSSION THREAD

**Discussion by Jme7** [community/community#22742] [New to GitHub]
<p>Is the Artic Code Vault ACTUALLY in a frozen warehouse?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Actually, no. It’s in a decommissioned coal mine:</p>
<div class="onebox lazyYT lazyYT-container" data-youtube-id="fzI9FNjXQ0o" data-youtube-title="GitHub Arctic Code Vault" data-parameters="feature=oembed&amp;wmode=opaque">
  <a href="https://www.youtube.com/watch?v=fzI9FNjXQ0o" target="_blank" rel="noopener">
    <img class="ytp-thumbnail-image" src="https://user-images.githubusercontent.com/1038121/181072772-40872e90-4b07-4a46-942c-f1579484cac8.jpeg" title="GitHub Arctic Code Vault" width="690" height="388">
  </a>
</div>

<p>But it’s in <a href="https://en.wikipedia.org/wiki/Svalbard">Svalbard, Norway</a>, so frozen would be pretty accurate :grinning:</p>

**Comment by Jme7** [comment]
<aside class="quote group-github-employees" data-username="lee-dohm" data-post="2" data-topic="125818">
<div class="title">
<div class="quote-controls"></div>
 lee-dohm:</div>
<blockquote>
<p>But it’s in <a href="https://en.wikipedia.org/wiki/Svalbard" rel="nofollow noopener">Svalbard, Norway</a>, so frozen would be pretty accurate</p>
</blockquote>
</aside>
<p>:grinning:</p>
<p>Ok, thanks!</p>
